src/js/ia/DDH.js

```diff
@@ -7,5 +7,20 @@
 
     };
 
+    env.Utils = {
+        // Called on userpages links click
+        sendReq: function($obj) {
```

Comment:
It might be clearer to others in the future if this were called `sendPixel`? Up to you, not worth holding up if you want to merge.